src/Compilers/VisualBasic/Portable/Errors/Errors.vb

```diff
@@ -1957,7 +1957,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         WRN_Experimental = 42380
 
         ' // AVAILABLE                             42381 - 49998
-        ERRWRN_Last = WRN_Experimental
+        ERRWRN_NextAvailable = 42381
```

Comment:
>42381 [](start = 31, length = 5)

If I understand correctly, the purpose of this sentinel/marker is to separate errors (below this) from other identifiers (above this). Why don't we set this to `49999`, so that we don't have to change the value again?
(We'd also want to pick another name)